pkg/packagekit/manifest.json

```diff
@@ -1,6 +1,11 @@
 {
     "name": "updates",
     "priority": 0,
+    "conditions": {
+        "path-exists": "/etc/PackageKit",
```

Comment:
This feels wrong.  I might be inclined to delete this path and still expect things to work (with the default config).  Even if that doesn't work today, it might change at some point in the future, possibly by way of alternate implementation of the same API.

Reply:
It wasn't my preferred choice either admittedly, but there actually aren't any other files with a predictable path between distros. Betwen /usr/libexec/ vs. /usr/lib/packagekit/ or /lib vs. /usr/lib there's an annoying amount of variation. However, `/lib/systemd/system/packagekit.service` actually ought to work, as even on /usr-merged systems there's always a /lib compat symlink anyway. Maybe we also eventually want `unit-exists:`? Asking systemd via d-bus is expensive, but we could just look into /lib/ vs. /usr/lib -- either way, it'd be a nice abstraction whose implementation we could change in the future. WDYT @allisonkarlitskaya ?

Reply:
Changed to test for the systemd unit.